bullet-featherstone/src/SDFFeatures.cc

```diff
@@ -1051,34 +1059,77 @@ bool SDFFeatures::AddSdfCollision(
          ++submeshIdx)
     {
       auto s = mesh->SubMeshByIndex(submeshIdx).lock();
-      auto vertexCount = s->VertexCount();
-      auto indexCount = s->IndexCount();
-      btAlignedObjectArray<btVector3> convertedVerts;
-      convertedVerts.reserve(static_cast<int>(vertexCount));
-      for (unsigned int i = 0; i < vertexCount; i++)
+      bool meshCreated = false;
+      if (meshSdf->Simplification() == "convex_decomposition")
```

Comment:
would it be possible to also handle the `convex_hull` case in this pull request? I would think that would be simpler than the `convex_decomposition` case, but would still be useful in cases where users know that a mesh is already convex

Reply:
yep I was going to add it as a followup PR but I'll bundle it together.

Reply:
addressed in new PR https://github.com/gazebosim/gz-physics/pull/606 that targets main. Supported by telling MeshManager to generate 1 convex hull. 

Alternative is to feed all vertices to `btConvexHullShape` and let bullet do the convex hull generation (intead of MeshManager) but that mean it'll be difficult for gz-sim visualize the generated collision. So I opted to let MeshManager create the convex hull instead so we can use the same function in the gui plugin to generate the collision visuals